src/core/webrtc/room-context.ts

```diff
@@ -66,6 +69,28 @@ interface ChatDataObject extends DataObject {
   metadata?: object | string
 }
 
+/**
```

Comment:
The comments on this are misleading

Reply:
@hudbuddy Thanks!. This is been updated.